common/persistence/cassandra/matching_task_store.go

```diff
@@ -39,6 +39,99 @@ import (
 	"go.temporal.io/server/common/primitives/timestamp"
 )
 
+const (
+	templateCreateTaskQuery = `INSERT INTO tasks (` +
+		`namespace_id, task_queue_name, task_queue_type, type, task_id, task, task_encoding) ` +
+		`VALUES(?, ?, ?, ?, ?, ?, ?)`
+
+	templateCreateTaskWithTTLQuery = `INSERT INTO tasks (` +
+		`namespace_id, task_queue_name, task_queue_type, type, task_id, task, task_encoding) ` +
+		`VALUES(?, ?, ?, ?, ?, ?, ?) USING TTL ?`
+
+	templateGetTasksQuery = `SELECT task_id, task, task_encoding ` +
+		`FROM tasks ` +
+		`WHERE namespace_id = ? ` +
+		`and task_queue_name = ? ` +
+		`and task_queue_type = ? ` +
+		`and type = ? ` +
+		`and task_id > ? ` +
+		`and task_id <= ?`
+
+	templateCompleteTaskQuery = `DELETE FROM tasks ` +
+		`WHERE namespace_id = ? ` +
+		`and task_queue_name = ? ` +
+		`and task_queue_type = ? ` +
+		`and type = ? ` +
+		`and task_id = ?`
+
+	templateCompleteTasksLessThanQuery = `DELETE FROM tasks ` +
+		`WHERE namespace_id = ? ` +
+		`AND task_queue_name = ? ` +
+		`AND task_queue_type = ? ` +
+		`AND type = ? ` +
+		`AND task_id <= ? `
+
+	templateGetTaskQueue = `SELECT ` +
```

Comment:
ends with Query to be consistent with others

Reply:
Updated.